templates/index.html

```diff
@@ -30,7 +29,7 @@
         <input type="name" id="txtName" name="name" class="form-control" placeholder="Name" required autofocus>
         <input type="tel" id="txtPhoneNumber" name="phoneNumber" class="form-control" placeholder="Phone#" required>
 
-        <button class="btn btn-lg btn-default btn-block" type="submit">Register</button>
+        <button class="btn btn-lg btn-block" type="button">Register</button>
```

Comment:
We need this type to be submit, as then it will add the name and phone number to the list. Was this tested? :)